src/style.css

```diff
@@ -1 +1,35 @@
 /* add styles here */
+
+/* okay */
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url('./images/star.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0),
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('./images/star-active.svg');
+}
```

Comment:
The use of the `:nth-child` pseudo-class to selectively apply styles is a smart way to handle variable states of stars. Ensure that the naming convention for modifiers (like `.stars--0`, `.stars--1`, etc.) is consistently followed throughout the project.